### PR TITLE
Update Getting-Started.md

### PR DESCRIPTION
**Pull request by jwolfe-coveo**

### CODE REVIEW

File: _docs/Getting-Started.md [hunk 1]
```diff
@@ -59,17 +57,31 @@ You can also change the styling of the component by changing the `src\stylesheet
 
 Once you have modified your component and are ready to test it, you need to run the build command, so the code can be built.
 
-To do so, run the following command, replacing `TestComponent` with the name of your component:
+To do so, run the following command, provided by the Makefile generated with the project:
```
Comment (amoreauCoveo) [inline _docs/Getting-Started.md]: I would change it to:
To do so, run the following command, provided by the Makefile generated with the project. Remember to replace `TestComponent` with the name of your component.

File: _docs/Getting-Started.md [hunk 2]
```diff
@@ -12,36 +12,34 @@ This article explains how to get started using Coveo Turbo to create, modify, an
...
+- Initializes a package.json file with all the necessary configuration and scripts.
+- Installs all dependencies needed by the CLI and the resulting project.
 - Create a component class with the appropriate template.
 - Create associated stylesheets to help style your component.
-- Create a sandbox environment that contains a basic Coveo search page, where you can test your component.
+- Create a basic Coveo search page, where you can test your component.
 - Create and fill a readme.md file with basic usage information, to help you have basic usage documentation available for each component.
 
-> The sandbox environment is not automatically connected to a Coveo Cloud organization; this can be added when serving the component.
+> The page is not automatically connected to a Coveo Cloud organization; this can be added when serving the component.
```
Comment (amoreauCoveo) [inline _docs/Getting-Started.md]: The search page

File: _docs/Getting-Started.md [hunk 3]
```diff
@@ -12,36 +12,34 @@ This article explains how to get started using Coveo Turbo to create, modify, an
 
 > For the full reference of the Coveo Turbo CLI, see [CLI Reference]({{ site.baseurl }}/docs/CLI-Reference).
 
-## Install Coveo Turbo
+## Install Coveo Turbo Prerequisites
```
Comment (amoreauCoveo) [inline _docs/Getting-Started.md]: I would rename this simply "Prerequisites"

File: _docs/Getting-Started.md [hunk 4]
```diff
@@ -12,36 +12,34 @@ This article explains how to get started using Coveo Turbo to create, modify, an
...
 To install Coveo Turbo, you need a Node environment with npm. You can download and install both on the [node.js website](https://nodejs.org/en/).
 
 > Coveo Turbo requires at least version 12 of Node.js. While using previous versions may work for certain commands, it's possible that other commands will not work properly.
 
-Once you have installed Node.js, you can run the following command to install and prepare the Coveo Turbo CLI:
-
-```
-npm install --save-dev @coveops/cli
-```
+Once you have installed Node.js, you can create a project and our CLI will take care of all the semantics for you.
```
Comment (amoreauCoveo) [inline _docs/Getting-Started.md]: You mean with the command coming after, right?
If that's the case, then just remove this line completely.

Comment (jwolfe-coveo) [reply _docs/Getting-Started.md]: Yep, ok I'll remove it

**Review by amoreauCoveo — APPROVED** [review]

**Pull request merged**